rosidl_runtime_py/rosidl_runtime_py/set_message.py

```diff
@@ -33,4 +38,13 @@ def set_message_fields(msg: Any, values: Dict[str, str]) -> None:
         except TypeError:
             value = field_type()
```

Comment:
I am surprised this works in cases where the field type is `numpy.ndarray` or `array.array`. Can you please also add a test with a static array (which would cover the `numpy` case).

Reply:
Yeah it surprised me as well that the code comparing to `list` didn't need to be updated after the numpy array PR..

Static array test case added in https://github.com/ros2/rosidl_python/pull/33/commits/712d4dafecad5fc5e416467a5f7d8a5f04b748dc